Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -110,6 +141,26 @@ void ABenchmarkGymGameMode::Tick(float DeltaSeconds)
 				}
 			}
 		}
+
+		for (int i = AIControlledPlayers.Num() - 1; i >= 0; i--)
+		{
+			if (AIControlledPlayers[i] == nullptr)
+			{
+				AIControlledPlayers.RemoveAt(i);
```

Comment:
RemoveAtSwap is quicker. Might also be nicer to just have a single array with both the Player and Idx.